template-parts/acf-featured-pages.php

```diff
@@ -0,0 +1,68 @@
+<?php
+// check that the ACF plugin is activated
+include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+if ( is_plugin_active( 'advanced-custom-fields-pro/acf.php' ) ) {
+
+		if( have_rows('featured_pages') ):
+
+				echo '<hr class="c-divider">';
+				echo '<div class="o-layout  o-layout--wide">';
+
+				// loop through the rows of data
+				while ( have_rows('featured_pages') ) : the_row();
+
+						echo '<div class="o-layout__item  u-4/12@lg">';
+
+						// page header with title link
+						if ( have_rows('page_header') ) :
```

Comment:
Could we remove this and use the Header row in the flex content area? Would add flexibility for other uses. Maybe we should introduce an option into Flex to allow for the headers to be linked? Is this a concept we want to use universally?

Reply:
Agreed and done. If a link is provided, it will be used to create a link. Otherwise the header gets output as normal. I also added the ability to specify header size.